modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -463,6 +465,18 @@ class BashWrapperBuilder {
         }
     }
 
+    static protected boolean isRetryable0(Exception e) {
+        if( e instanceof FileSystemException )
+            return true
+        if( e instanceof SocketException )
+            return true
+        if( e instanceof RuntimeException )
+            return true
+        if( e.class.getSimpleName() == 'HttpResponseException' )
```

Comment:
```suggestion
        if( e.class.getName() == 'com.google.api.client.http.HttpResponseException' )
```

Reply:
Actually, I did `HttpResponseException` on purpose. The logic is that any http prefixed class name is eligible for retry

Reply:
okay 🤞